### PR TITLE
🧹 Clean up api client after the TRIM

### PR DESCRIPTION
**Pull request by boredland**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,20 +75,20 @@ npm run generate
 
 This will access all OpenAPI endpoints and generate changes based on changes to the API.
 
-2. Check if `src/index.ts` needs to be adjusted. In case of bigger changes, for example changes to the method of an endpoint or a new endpoint, you need to update the corresponding methods.
+1. Check if `src/index.ts` needs to be adjusted. In case of bigger changes, for example changes to the method of an endpoint or a new endpoint, you need to update the corresponding methods.
```
Comment (jeysal) [inline README.md:78]: o.O there's two `1.` now

**Review by jeysal — COMMENTED** [review]
did you link into `rolling-rick` yet to see if that works?

File: README.md (hunk 1 above)
Comment (boredland) [reply README.md]: oopsie.

**Comment by boredland** [comment]
> did you link into `rolling-rick` yet to see if that works?

No, but I had a brief look and don't think we use the "live" functions anywhere.

**Comment by jeysal** [comment]
okay I did and it seems 2 work

**Review by jeysal — APPROVED** [review]
ThE tRiM

**Pull request merged**